client/src/partials/templates/grid/voucherType.tmpl.html

```diff
@@ -1,5 +1,5 @@
 <div style="padding: 5px;">
-  <span ng-class="{'label label-success': grid.appScope.getType(row.entity.type_id).incomeExpense == 'income', 'label label-warning': grid.appScope.getType(row.entity.type_id).incomeExpense == 'expense'}" href="">
+  <span ng-class="{'label label-success': grid.appScope.getType(row.entity.type_id).type == 'income', 'label label-warning': grid.appScope.getType(row.entity.type_id).type == 'expense'}" href="">
```

Comment:
Can you make the `getType()` method return the `.type` property, or would that disrupt some other code?  It seems to me cleaner to say
```js
grid.appScope.getType(row.entity.type_id) == 'expense'
```
rather than
```js
grid.appScope.getType(row.entity.type_id).type == 'expense'
```